code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -0,0 +1,314 @@
+
+
+//Dismember a limb
+/obj/item/bodypart/proc/dismember()
+	var/mob/living/carbon/human/H = owner
+	if(!istype(H) || !H.dna.species.has_dismemberment) // species don't allow dismemberment
+		return 0
+
+	var/obj/item/bodypart/affecting = H.get_bodypart("chest")
+	affecting.take_damage(Clamp(brute_dam/2, 15, 50), Clamp(burn_dam/2, 0, 50)) //Damage the chest based on limb's existing damage
+	H.visible_message("<span class='danger'><B>[H]'s [src] has been violently dismembered!</B></span>")
+	H.emote("scream")
+	drop_limb()
+
+	var/direction = pick(cardinal)
+	var/t_range = rand(2,max(throw_range/2, 2))
+	var/turf/target_turf = get_turf(src)
+	for(var/i = 1; i < t_range; i++)
+		var/turf/new_turf = get_step(target_turf, direction)
+		target_turf = new_turf
+		if(new_turf.density)
+			break
+	throw_at_fast(target_turf, throw_range, throw_speed)
```

Comment:
When you want to drop a limb without the screaming and throwing, you use "drop_limb()".